src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -902,6 +902,54 @@ class RBDService : public ServiceBase {
     }
 };
 
+class WNBDWatchCtx : public librbd::UpdateWatchCtx
+{
+private:
+  PWNBD_DISK disk;
```

Comment:
let's reorder those members so that we avoid the warning:

```
/home/jenkins-build/build/workspace/ceph-windows-pull-requests/ceph/src/tools/rbd_wnbd/rbd_wnbd.cc: In constructor 'WNBDWatchCtx::WNBDWatchCtx(librados::v14_2_0::IoCtx&, WnbdHandler*, librbd::Image&, uint64_t)':
/home/jenkins-build/build/workspace/ceph-windows-pull-requests/ceph/src/tools/rbd_wnbd/rbd_wnbd.cc:910:20: warning: 'WNBDWatchCtx::io_ctx' will be initialized after [-Wreorder]
  910 |   librados::IoCtx &io_ctx;
      |                    ^~~~~~
/home/jenkins-build/build/workspace/ceph-windows-pull-requests/ceph/src/tools/rbd_wnbd/rbd_wnbd.cc:909:16: warning:   'WnbdHandler* WNBDWatchCtx::handler' [-Wreorder]
  909 |   WnbdHandler* handler;
      |                ^~~~~~~
/home/jenkins-build/build/workspace/ceph-windows-pull-requests/ceph/src/tools/rbd_wnbd/rbd_wnbd.cc:914:3: warning:   when initialized here [-Wreorder]
  914 |   WNBDWatchCtx(librados::IoCtx &_io_ctx,
      |   ^~~~~~~~~~~~
```